[2024-05-20T20:05:33.087+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-20T20:05:33.247+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.call_api manual__2024-05-20T20:05:22.163014+00:00 [queued]>
[2024-05-20T20:05:33.313+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.call_api manual__2024-05-20T20:05:22.163014+00:00 [queued]>
[2024-05-20T20:05:33.314+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-20T20:05:33.436+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): call_api> on 2024-05-20 20:05:22.163014+00:00
[2024-05-20T20:05:33.486+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=94) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-20T20:05:33.573+0000] {standard_task_runner.py:63} INFO - Started process 95 to run task
[2024-05-20T20:05:33.531+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'call_api', 'manual__2024-05-20T20:05:22.163014+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpgz_4l14s']
[2024-05-20T20:05:33.580+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask call_api
[2024-05-20T20:05:33.760+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.call_api manual__2024-05-20T20:05:22.163014+00:00 [running]> on host 1c53b53fc77d
[2024-05-20T20:05:34.107+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='call_api' AIRFLOW_CTX_EXECUTION_DATE='2024-05-20T20:05:22.163014+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-20T20:05:22.163014+00:00'
[2024-05-20T20:05:34.110+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-20T20:05:34.334+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-20T20:05:34.336+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-20T20:05:34.360+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=call_api, run_id=manual__2024-05-20T20:05:22.163014+00:00, execution_date=20240520T200522, start_date=20240520T200533, end_date=20240520T200534
[2024-05-20T20:05:34.404+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-20T20:05:34.470+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-20T20:05:34.478+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
